lib/factory_girl/strategy/stub.rb

```diff
@@ -55,6 +55,14 @@ def update_attribute(*args)
           def update_column(*args)
             raise "stubbed models are not allowed to access the database - #{self.class.to_s}#update_column(#{args.join(",")})"
           end
+
+          def increment!(*args)
+            raise "stubbed models are not allowed to access the database - #{self.class.to_s}#increment!(#{args.join(",")})"
```

Comment:
Redundant use of `Object#to_s` in interpolation.<br>Line is too long. [124/80]<br>Prefer single-quoted strings inside interpolations.